lib/appcontroller_client.py

```diff
@@ -55,6 +56,14 @@ class AppControllerClient():
   # The number of times we should retry SOAP calls in case of failures.
   DEFAULT_NUM_RETRIES = 5
 
+  # The initial amount of time we should sleep when waiting for UserAppServer
+  # metadata to change state.
+  STARTING_SLEEP_TIME = 1
```

Comment:
I believe we are moving away from this concept of increasing the timeout. 
It is probably a good idea to remove this and use a fixed timeout on each iteration.